cmd/edit.go

```diff
@@ -70,7 +70,12 @@ func EditFile(cmd *cobra.Command, args []string) error {
 
 	fileEditor := editor.NewEditor(func(modified string) error {
 		file.Contents = modified
-		return DefaultAPIClient.PutAuroraConfigFile(file, eTag)
+
+		// Save config file (Gobo)
+		if _, err = DefaultAPIClient.UpdateAuroraConfigFile(file, eTag); err != nil {
+			return err
```

Comment:
Se tidligere kommentar

Reply:
Skal se nøyere på feilhåndtering.  Mulig det blir en egen Jirasak, slik at vi kan fikse det i ao generelt.

Reply:
Egen Jirasak opprettet for dette.